elixir-sdk/native/sdk_core/src/lib.rs

```diff
@@ -29,6 +30,7 @@ const SDK_METADATA: SdkMetadata = SdkMetadata {
 pub struct EppoClient {
     pub evaluator: Evaluator,
     pub background_thread: BackgroundThread,
+    pub configuration_poller: Option<ConfigurationPoller>,
```

Comment:
Given we always have a poller, does this field needs to be an `Option`?

related: do these fields need to be public? Just want to confirm that they are not part of our public API and we can change them later